tools/check_man.py

```diff
@@ -182,11 +184,15 @@ def parse_cmdline():
                         help="man-page in mdoc format")
     parser.add_argument("-c", "--check-file",
                         help="check that all options are in the file")
+    parser.add_argument("--edit-bash",
```

Comment:
Should "edit" be "regenerate"?

Reply:
Hmm.  It certainly doesn't generate it from scratch, but I suppose that "regenerate" could work.

Reply:
... or maybe "update", since that's what I ended up calling the branch?